test/scripts/e2e_subs/goal/goal.py

```diff
@@ -240,21 +240,21 @@ def finish(self, tx, send):
         return tx
 
     def keyreg(self, sender, votekey=None, selkey=None, votefst=None,
-               votelst=None, votekd=None,
+               votelst=None, votekd=None, sprfkey=None,
                send=None, **kwargs):
-        params = self.algod.suggested_params()
+        params = self.params(kwargs.pop("lifetime", 1000), kwargs.pop("fee", None))
```

Comment:
Correct. In my little testing library, `goal.py` I find it nicer to specify a lifetime rather than specify an `fv` and `lv` round,. (It uses the suggested `fv` and computes `lv`)
```
    def params(self, lifetime=None, fee=None):
        params = self.algod.suggested_params()
        if lifetime is not None:
            params.last = params.first + lifetime
```